nav-menu-roles.php

```diff
@@ -382,7 +382,7 @@ public function nav_update( $menu_id, $menu_item_db_id ) {
 			}
 			if ( ! empty ( $custom_roles ) ) $saved_data = $custom_roles;
 		} else if ( isset( $_POST['nav-menu-logged-in-out'][$menu_item_db_id]  )  && in_array( $_POST['nav-menu-logged-in-out'][$menu_item_db_id], array( 'in', 'out' ) ) ) {
```

Comment:
@grappler this line already tests that the value is either 'in' or 'out'. Anything else won't meet the IF condition and won't be saved. Therefore since the value is already restricted to 2 options the text field is redundant. Does that make sense?